docs_2.0/manual_EN/2.introduction/1.what-is-nebula-graph.md

```diff
@@ -0,0 +1,69 @@
+# What is Nebula Graph
+
+Nebula Graph is an open-source, distributed, easily scalable, and native graph database. It is capable of hosting graphs with billions of vertices and trillions of edges, and serve queries with millisecond-latency.
+
+## What is a graph database
+
+A graph database, such as Nebula Graph, is a storage engine that specializes in storing and retrieving vast networks of information. It efficiently stores data as vertices (nodes) and edges (relationships) and allows high-performance retrieval and querying of those structures. Properties can be added to both vertices and edges. Vertices can have zero to multiple tags, and edges are always directed and typed.
```

Comment:
Properties can be added --attached -- to both vertices and edges. Vertices can have zero -- at least one -- to multiple tags, and edges are always directed and typed.